multiqc/utils/search_patterns.yaml

```diff
@@ -4,9 +4,11 @@
 
 adapterRemoval:
     fn: '*.settings'
+    shared: true
 afterqc:
     fn: '*.json'
     contents: 'allow_mismatch_in_poly'
+    shared: true
```

Comment:
This shouldn't be needed, as it also needs the string `allow_mismatch_in_poly` in the file which should be specific enough (it's not claiming all JSON files).